__tests__/app.test.js

```diff
@@ -129,4 +129,48 @@ describe('/api/articles', () => {
             expect(message).toBe('Endpoint Not Found')
         })
     })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET 200: Should return all the comments for a given article', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body: { comments }}) => {
+            expect(comments.length).toBe(11)
+            comments.forEach((comment) => {
+                expect(comment).toHaveProperty('comment_id')
+                expect(comment).toHaveProperty('votes')
+                expect(comment).toHaveProperty('created_at')
+                expect(comment).toHaveProperty('author')
+                expect(comment).toHaveProperty('body')
+                expect(comment).toHaveProperty('article_id')
+            })
```

Comment:
Check out the jest matcher [toMatchObject](https://jestjs.io/docs/expect#tomatchobjectobject)

An example of how it can be used:
`expect(person).toMatchObject({
  name: expect.any(String),
  age: expect.any(Number)
}`

I think it looks a bit cleaner than having `toHaveProperty` repeated out over a few lines and will show off a bit of jest skillset to employers ✨